OpenRA.Game/Map/MapCache.cs

```diff
@@ -150,9 +150,10 @@ public void QueryRemoteMapDetails(IEnumerable<string> uids, Action<MapPreview> m
 					foreach (var kv in yaml)
 						maps[kv.Key].UpdateRemoteSearch(MapStatus.DownloadAvailable, kv.Value, mapDetailsReceived);
 				}
-				catch
+				catch (Exception e)
 				{
 					Log.Write("debug", "Can't parse remote map search data:\n{0}", data);
+					Log.Write("debug", "Exception: {0}", e);
```

Comment:
Not a job for this PR, but we should think about redirecting these messages to server.log on dedicated servers instead.